.gitignore

```diff
@@ -1,2 +1,3 @@
 # Temporary generated files
-_out
\ No newline at end of file
+_out
+.idea
```

Comment:
what's .idea?

Reply:
the project folder from intellij IDEA (or goland).